models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
The idea is that we can eventually link to other models that aren't `Team`. For example, a lot of the New England district events have their own twitter profiles (example: https://twitter.com/wtbydistrict). So it'll be cleaner, I think, to keep one join model that can reference multiple types of parent